src/cli.ts

```diff
@@ -73,6 +73,21 @@ const handleJobExecution = (input: string, _: unknown) => {
   return input;
 };
 
+const handleOnboardingType = (input: string, _: unknown) => {
+  if (input === 'preconnect') {
+    return 'preconnect';
+  }
+
+  if (input === 'jitp') {
+    return input;
+  }
+
+  new Log(false).error(
+    'Input for associate must be blank, "jitp" or "preconnect"',
+  );
+  process.exit();
```

Comment:
I'm picking nits but the returns are inconsistent here, line 78 returns a hardcoded string, while L82 returns the actual input, I would like to see that be consistent. Also, adding a `process.exit(1);` is just good form

could also be managed something like this way. 

```suggested
  const validInputs = ['preconnect', 'jitp'];

  if (validInputs.includes(input)) {
    return input;
  }

  new Log(false).error(
    'Input for onboarding type must be "jitp" or "preconnect"',
  );
  process.exit(1);
```

Reply:
Good catch